thinkbox-deadline/mainTemplate.json

```diff
@@ -21,6 +21,15 @@
             }
         },
 
+        "clusterName":

```

Comment:
Can you rename this file back to azuredeploy.json? The rename breaks our "Deploy to Azure" experience and the validation bot.